receiver/dronereceiver/README.md

```diff
@@ -0,0 +1,166 @@
+# Drone Receiver
+
+<!-- prettier-ignore-start -->
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [development]: logs, traces, metrics   |
+| Distributions | [grafana-ci-otel-collector] |
+| Issues        | [![Open issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aopen%20label%3Areceiver%2Fdrone%20&label=open&color=orange&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aopen+is%3Aissue+label%3Areceiver%2Fdrone) [![Closed issues](https://img.shields.io/github/issues-search/open-telemetry/opentelemetry-collector-contrib?query=is%3Aissue%20is%3Aclosed%20label%3Areceiver%2Fdrone%20&label=closed&color=blue&logo=opentelemetry)](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aclosed+is%3Aissue+label%3Areceiver%2Fdrone) |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@Elfo404](https://www.github.com/Elfo404) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[grafana-ci-otel-collector]: 
+<!-- end autogenerated section -->
+<!-- prettier-ignore-end -->
+
+The Drone Receiver generates metrics, traces, and logs from the Drone CI/CD system.
+
+For a list of generated metrics, see the [generated metrics documentation](./documentation.md).
+
+## Configuration
+
+> [!TIP]  
+> The following configuration should work OOTB with the docker compose file provided.
+
+```yaml
+receivers:
+  dronereceiver:
+    collection_interval: 1m
+    path: /drone/webhook
+    endpoint: localhost:3333
+    secret: bea26a2221fd8090ea38720fc445eca6
+    drone:
+      token: <DRONE_TOKEN>
+      host: http://localhost:8080
+      database:
+        host: localhost
+        db: drone
+        username: postgres
+        password: postgres
+    repos:
+      org1/repo1:
+        - main
+      org1/repo2:
+        - main
+        - development
+      org2/another-repo:
+        - main
+        - v1.0.x
+```
+
+## Local Drone instance
+
+It is possible to use a local Drone instance for easier development.
+
+> [!NOTE]  
+> Note that by default no webhook events are sent to the receiver from GitHub (i.e. when
+> pushing to a branch to trigger a build), so make sure your pipelines can be run manually.
+>
+> If you want to trigger pipelines in your local Drone instance when pushing, you need to setup
+> webhooks in your repository. Refer to Drone docs for more informations.
```

Comment:
Nit: Link?

Reply:
couldn't find the docs on drone website 🤷 

added instructions in f35a611